framework/midi/internal/synthesizersetup.cpp

```diff
@@ -16,18 +16,32 @@
 //  along with this program; if not, write to the Free Software
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
-
-#include "sffproviderlocalfile.h"
+#include "synthesizersetup.h"
 
 #include "log.h"
 
 using namespace mu::midi;
 
-void SFFProviderLocalFile::loadSF(const midi::Programs& programs, const OnLoading& onloading, const OnLoaded& onloaded)
+void SynthesizerSetup::setup()
```

Comment:
I propose to add a callback to AudioEngine::init and make setup of the synthesizer in it

Reply:
The audio engine is in one module that knows nothing about the synthesizer. The synthesizer is in another module, which does not call the audio engine init